packages/@okta/i18n/src/properties/login.properties

```diff
@@ -723,6 +723,7 @@ oie.password.enroll.title = Set up password
 oie.password.passwordLabel = Enter password
 oie.password.confirmPasswordLabel = Re-enter password
 oie.password.reset.verification = Verify with one of the following authenticators to reset your password.
+oie.password.reset.username.or.email.placeholder = Username or email
```

Comment:
we are calling them `label` in oie instead of `placeholder`.. no rule for this but we can have consistency in naming.

Reply:
Removed this change. Using an existing string